config/sync/core.extension.yml

```diff
@@ -6,10 +6,12 @@ module:
   admin_toolbar_tools: 0
   bixal_default_content: 0
   block: 0
+  block_node: 0
   breakpoint: 0
   ckeditor5: 0
   components: 0
   contextual: 0
+  ctools: 0
```

Comment:
I see you added paragraphs_blocks as a dependency but only ctools (paragraphs_blocks dependency) is enabled. 
There's also no composer.log entry for it. 
Did you mean to work with either of these modules?

Reply:
Sorry I missed this comment Matt, need to change my github to my company email because it's still locked to my personal. I enabled both of these modules when I was messing around in a branch testing out layout builder module, and when I made this branch to extrapolate the hero work via cherry-pick I missed cleaning these two modules from config. I can fix that with my other changes I push up.

Reply:
@mpbixal this branch has been updated and refactored, I also disabled ctools and removed drupal paragraph blocks.